perun-ldapc/src/main/java/cz/metacentrum/perun/ldapc/processor/impl/FacilityAttributeProcessor.java

```diff
@@ -21,9 +25,10 @@
 	private static Pattern facilitySetPattern = Pattern.compile(" set for Facility:\\[(.*)\\]");
 	private static Pattern facilityRemovePattern = Pattern.compile(" removed for Facility:\\[(.*)\\]");
 	private static Pattern facilityAllAttrsRemovedPattern = Pattern.compile("All attributes removed for Facility:\\[(.*)\\]");
+	private static Pattern facilityVirtualChangePattern = Pattern.compile(" changed for Facility:\\[.*\\]");
```

Comment:
Please change ``.*`` -> ``(.*)``. Its not necessary, but I like all regexes looking the same at first sight, so that you can focus on rest of regex content, if its not working properly.